testing/src/main/java/io/stargate/it/http/MetricsTest.java

```diff
@@ -67,12 +73,14 @@ public void restApiHttpRequestMetrics() throws IOException {
 
     String result = RestUtils.get("", String.format("%s:8084/metrics", host), HttpStatus.SC_OK);
 
-    String[] lines = result.split(System.getProperty("line.separator"));
+    List<String> lines = Arrays.stream(result.split(System.getProperty("line.separator")))
+            .filter(line -> line.startsWith("http_server_requests"))
+            .collect(Collectors.toList());
```

Comment:
nit: this is repeated a few times, maybe you could extract a `filter(String[] lines, String prefix)` method.

Reply:
So I tend not to extract common code for testing, as:

* reading a single test becomes more complicated, as you need to navigate to "helper" methods in order to realize what's going on
* changes in the extracted peaces start to influence too many tests
* of course this does not stand true for the some INT test setup for example

Would it be OK to leave it like this?